docs/ldml/tr35-keyboards.md

```diff
@@ -2540,7 +2555,28 @@ content-char
            | DIGIT
            | NON-ASCII
 escaped-char
-         ::= '\' ( '\' | '{' | '|' | '}' )
+         ::= '\' escapable-char
+escapable-char
+         ::= '\'
```

Comment:
This order is not the same as the spec order. Might be good if they matched up and were in character value order